source/_integrations/lamarzocco.markdown

```diff
@@ -177,6 +179,8 @@ mode: single
 - Only La Marzocco native app accounts are supported, social logins (Google, Apple & WeChat) are not supported
 - Currently it is only possible to view the schedules configured in the La Marzocco Home app, but not to edit the schedules from Home Assistant. You can, of course, build Home Assistant native automations to reflect the same functionality in Home Assistant.
 
+{% include common-tasks/remove_device_service.md %}
```

Comment:
@c0ffeeca7 apologies for the ping, but would your rather have this used here, or just copy the 3 steps out of it? Because having `Depending on the integration, additional steps can be needed, such as resetting the device or to delete credentials. Refer to the integration documentation to see if additional steps are needed.` on the integration page is also a bit awkward...

Reply:
FYI: https://github.com/home-assistant/home-assistant.io/pull/35816

Reply:
Sounds good 👍